### ISSUE TITLE
Extend zesFabricPortGetThroughput() to allow collecting data from multiple ports in a single call

### ISSUE THREAD

**Issue by matcabral** [oneapi-src/level-zero-spec#85]
To optimize performance and avoid multiple expensive system calls while collecting throughput it would be much beneficial to allow the API collect throughput on a list of ports in a single call. During application runtime, live throughput collection is the most time sensitive operation during which performance tools will pursue the least overhead. With the current API signature, there is no room for improvement and the tool must call once per port. Allowing multiple ports data to be collected in a single call would allow implementations to optimize the number of calls through the entire stack. 
Proposed API: 

```
 ze_result_t ZE_APICALL zesFabricPortGetThroughput(uint16_t numPorts, zes_fabric_port_handle_t *hPort, zes_fabric_port_throughput_t *pThroughput)
```
where the ports handles and throughput data become pointers to arrays allocated by the user.

**Comment by saik-intel** [comment]
@wdamon-intel  @jandres742   please assign this spec issue to Sanil and we would like to have this change ready in SPEC 1.7

**Comment by jandres742** [comment]
@sanilkumar0 please see above.

**Comment by sanilkumar0** [comment]
Problem is zes_fabric_port_handle_t  is enumerated per port. We may need change -zesDeviceEnumFabricPorts to enumerate all ports instead of one.

**Comment by matcabral** [comment]
Hi @sanilkumar0, zesDeviceEnumFabricPorts() does effectively enumerate multiple ports in a single call. Would you please elaborate on your comment? 
Besides the enumeration, the request for optimization in zesFabricPortGetThroughput() is independent of how ports are enumerated. Expectation is that ports are enumerated once, but querying throughput will be done repeatedly at fast intervals by monitoring and/or perf optimization tools. Therefore, this optimization is independent of enumeration.

**Comment by sanilkumar0** [comment]
Thanks.. I have created a PR to address this for 1.7 release - 
https://github.com/oneapi-src/level-zero-spec/pull/134

**Comment by wdamon-intel** [comment]
Closed with PR #134
